WORKSPACE

```diff
@@ -82,6 +82,30 @@ gogoslick_proto_library(
     verbose = 0,
 )
 
+# We cannot use gogoslick because it renames enums, causing a conflict between the enums declared in metric and label.
+gogo_proto_library(
+    name = "google/api",
```

Comment:
You don't need this gogo_proto_library entry, since the code doesn't use the generated pb.go files at all right now (genproto package is used instead).

I locally tried to remove this, mixs still compiled. FYI.